Source/JavaScriptCore/bytecode/InlineCacheCompiler.cpp

```diff
@@ -4240,10 +4240,12 @@ bool InlineCacheCompiler::canEmitIntrinsicGetter(StructureStubInfo& stubInfo, JS
         return false;
 
     switch (getter->intrinsic()) {
+    case DataViewByteLengthIntrinsic:
```

Comment:
Maybe separating clauses would make things easier to add other DataView function supports.
```
case DataViewByteLengthIntrinsic: {
    if (structure->typeInfo().type() != DataViewType)
        return false;
#if USE(JSVALUE32_64)
    if (isResizableOrGrowableSharedTypedArrayIncludingDataView(structure->classInfoForCells()))
        return false;
#endif
    return true;
}
```